src/core/auth.ts

```diff
@@ -54,23 +55,66 @@ export interface AuthSDK {
   verifyToken: (token: string) => Promise<FetchResult<VerifyTokenMutation>>;
 }
 
+type AuthKey = keyof AuthSDK;
+
 export const auth = ({
   apolloClient: client,
   channel,
 }: SaleorClientMethodsProps): AuthSDK => {
+  const authenticatingMutationStatus: Record<string, boolean> = {
+    login: false,
+    refreshToken: false,
```

Comment:
I don't think `refreshToken` is necessary, I'd keep it to the initial login methods (login, verifyToken and finnaly oidc logic)